qa/rolling-upgrade/src/test/resources/rest-api-spec/test/old_cluster/20_date_range.yml

```diff
@@ -1,9 +1,5 @@
 ---
 "Create index with joda style index that is incompatible with java.time. (6.0)":
-  - skip:
-      features: "allowed_warnings"
-      version: "6.8.1 -"
-      reason: change of warning message
```

Comment:
This one needs to keep the `allowed_warnings` feature: https://build.ci.opensearch.org/job/gradle-check/55731/testReport/junit/org.opensearch.upgrades/UpgradeClusterClientYamlTestSuiteIT/initializationError/

As is, since this one should be skipped on all versions newer than 6.8.1, I think we should remove the test altogether.